client/package-lock.json

```diff
@@ -15325,6 +15325,22 @@
         }
       }
     },
+    "react-contenteditable": {
```

Comment:
@robinv85 Do we need to add this file on this PR? I am not sure, I see the package.json file being updated so perhaps yes? @BMariscal I did not see any usages of this package, but maybe I am missing that part.

Reply:
yes there is a new dependency    "react-contenteditable": "^3.3.4",